### DISCUSSION TITLE
V2 and Custom component

### DISCUSSION THREAD

**Discussion by vicenterusso** [beholdr/maska#104] [Q&A]
Hi,

I'm already using successfully v1 with [tusen-ai/naive-ui](https://github.com/tusen-ai/naive-ui).

I also understand that V2 lacks this V1 feature:

- Works on any input (custom or native)

Question: Is there any workaround to make it work?

**Comment by beholdr** [comment]
I’ve made some fixes to work with `v-model` in v2.1. So it should better work with custom components. Please try it.
